ibacm/src/acm.c

```diff
@@ -2144,7 +2144,7 @@ static int acm_assign_ep_names(struct acmc_ep *ep)
 			type = ACM_ADDRESS_IP6;
 		} else {
 			type = ACM_ADDRESS_NAME;
-			memcpy(addr, name, strlen(name));
+			strncpy((char *)addr, name, sizeof(addr));
```

Comment:
Lets use snprintf, and why the cast?

Reply:
Why snprintf()? And the cast is because addr is uint8_t addr[ACM_MAX_ADDRESS] and
strncmp() expects the first arg to be a "char *". So I was getting a warning on compile.

Reply:
So it is always NULL terminated

Reply:
OK. But it is already always NULL terminated since name is initialized with the
sscanf() %s format earlier in the function.

Reply:
Oh? okay. lets leave it as strncpy()